vllm/_custom_ops.py

```diff
@@ -416,32 +416,43 @@ def scaled_fp8_quant(
 
 # int8
 def scaled_int8_quant(
-        input: torch.Tensor,
-        scale: Optional[torch.Tensor] = None
-) -> Tuple[torch.Tensor, torch.Tensor]:
+    input: torch.Tensor,
+    scale: Optional[torch.Tensor] = None,
+    azp: Optional[torch.Tensor] = None,
+    symmetric: bool = True
```

Comment:
can `symmetric` be inferred from the optional azp ?

Reply:
Unfortunately I don't think so. In the static case, yes, but in the dynamic case, neither scales nor azp are passed in both symmetric and asymmetric cases. I guess we could pass a dummy azp in the static asymmetric case but this felt cleaner.